public/app/plugins/datasource/cloud-monitoring/types.ts

```diff
@@ -126,6 +126,7 @@ export interface MetricQuery extends BaseQuery {
   view?: string;
   query: string;
   preprocessor?: PreprocessorType;
+  graphPeriod?: string;
```

Comment:
what do you think about graphPeriod: 'disabled' || string or something like that? I could see someone editing this in the future and sending along an empty string thinking they've disabled it when they haven't.